mitosheet/src/mito/components/taskpanes/Graph/graphUtils.tsx

```diff
@@ -584,37 +593,47 @@ export const registerClickEventsForGraphElements = (graphOutput: GraphOutput, se
     xtitle.style.pointerEvents = 'all'
     ytitle.style.pointerEvents = 'all'
 
+    /**
+     * Open context menu when right clicked
+     */
+    gtitle.addEventListener('contextmenu', (e: any) => {
+        e.preventDefault();
+        setSelectedGraphElement(getGraphElementInfoFromHTMLElement(gtitle, 'gtitle', graphOutput, 'context-menu'))
```

Comment:
These string literals are pretty hard to parse at first glance. Let's make them a little more descriptive. What does `none` even mean -- how can you set the selected graph element to none, is that different than setting it undefined?